### DISCUSSION TITLE
unexpected pointer type

### DISCUSSION THREAD

**Discussion by boojongmin** [99designs/gqlgen#2178] [Q&A]
### What happened?
```graphql
    field(id: ID!): [String!]! @goField(forceResolver: true)
```

```go
func (r *queryResolver) field(ctx context.Context, obj *mg.Query, ID int64) ([]*string, error) {
// ...
}
```

### What did you expect?

I expected return type signature is `[]string`

### versions
 - `go run github.com/99designs/gqlgen version`? `0.17.5`
 - `go version`? `go version go1.18 darwin/arm64`

thank you

**Comment by ryanli-me** [comment]
i have the same issue too

**Comment by chriskolenko** [comment]
Any updates on this?

**Reply by chriskolenko** [reply]
Found a solution to my problem. 

Added 
```
omit_slice_element_pointers: true 
```

To my gqlgen.yml file
